dashboard/app/controllers/reference_guides_controller.rb

```diff
@@ -0,0 +1,15 @@
+class ReferenceGuidesController < ApplicationController
+  include CurriculumHelper
+  authorize_resource
+
+  before_action :require_levelbuilder_mode_or_test_env
```

Comment:
We probably don't want to require levelbuilder mode for show but we will for edit and update. Maybe add `, except [:show]` to the end of this line